### PR TITLE
[XPU][PHI Kernels] bind rsqrt、bitwise_or、arange_tensor for xpu

### PR DESCRIPTION
**Pull request by lj970926**
### PR types

New features
### PR changes

OPs
### Description

1. bind rsqrt and bitwise_or kernels for xpu
2. fix bugs caused by the name changing in previous PR for arange.
3. replate the CPU implementation of XPU arange_tensor Kernel with xpu::range

### CODE REVIEW

File: paddle/phi/backends/xpu/xpu2_op_list.cc [hunk 1]
```diff
@@ -43,6 +43,10 @@ XPUOpMap& get_kl2_ops() {
       {"adagrad", XPUKernelSet({phi::DataType::FLOAT32})},
       {"addcmul_xpu",
        XPUKernelSet({phi::DataType::FLOAT32, phi::DataType::FLOAT16})},
+      {"arange_tensor",
+       XPUKernelSet({phi::DataType::FLOAT32,
+                     phi::DataType::INT32,
+                     phi::DataType::INT64})},
```
Comment (lj970926) [inline paddle/phi/backends/xpu/xpu2_op_list.cc:49]: https://github.com/PaddlePaddle/Paddle/pull/58381 这个PR把kernel的名字从arange改成了arange_tensor，导致xpu不能从xpu2/3_op_list中找到对应的kernel信息从而触发fallback

File: paddle/phi/kernels/xpu/arange_kernel.cc [hunk 2]
```diff
@@ -32,19 +33,11 @@ void ArangeTensorKernel(const Context& dev_ctx,
...
-  for (int64_t i = 0; i < size; ++i) {
-    out_cpu_data[i] = value;
-    value += step_value;
-  }
-  phi::Copy(dev_ctx, out_cpu, out->place(), true, out);
+  auto* out_data = dev_ctx.template Alloc<T>(out);
+
+  int ret = xpu::range<T>(
+      dev_ctx.x_context(), out_data, start_value, step_value, size);
+  PADDLE_ENFORCE_XDNN_SUCCESS(ret, "range");
```
Comment (lj970926) [inline paddle/phi/kernels/xpu/arange_kernel.cc:40]: 原来的CPU实现替换为xpu::range

File: paddle/phi/kernels/xpu/bitwise.cc [hunk 3]
```diff
@@ -44,7 +44,17 @@ void BitwiseAndKernel(const Context& ctx,
...
 }
+
+template <typename T, typename Context>
+void BitwiseOrKernel(const Context& ctx,
+                     const DenseTensor& x,
+                     const DenseTensor& y,
+                     DenseTensor* out) {
+  // Same reason as bitwise_and
+  LogicalOrKernel<T, Context>(ctx, x, y, out);
+}
```
Comment (lj970926) [inline paddle/phi/kernels/xpu/bitwise.cc:55]: bitwise_or之前已有已有单测

**Review by houj04 — APPROVED** [review]

**Review by runzhech — COMMENTED** [review]
lgtm

**Review by QingshuChen — APPROVED** [review]

**Pull request merged**